tasks/config/routes.rb

```diff
@@ -1,10 +1,14 @@
 Rails.application.routes.draw do
-  get '/' => 'tasks#index'
+  root :to => 'tasks#index'
+  # get '/' => 'tasks#index'
   get 'tasks/' => 'tasks#index'
   get 'tasks/new' => 'tasks#new'
   get 'tasks/:id' => 'tasks#show'
   post 'tasks/' => 'tasks#create'
   delete 'tasks/:id' => 'tasks#destroy'
+  patch 'tasks/:id' => 'tasks#update', :as => :task
```

Comment:
We'll be learning about "named routes" (what you get by using `as: :whatever`) later this week, but for now I think it's probably simplest to avoid using it if possible.